### ISSUE TITLE
Better UI for different versions of `MeetingDocument`s

### ISSUE THREAD

**Issue by tomudding** [GEWIS/gewisweb#1762]
### What would you like?

Add the option to be able to upload a new version of a specific `MeetingDocument`. This should be done through an additional button on the existing documents.

When this happens, the new document will have its `previousVersion` attribute (to be added) populated with the `id` of the previous `MeetingDocument` or the previous version links to the new version (such that the attribute on the new version is `NULL`; this may make querying easier).

In the overview, the newest version will be show by default and a dropdown should be added to be able to see previous versions.

### Why is this needed?

History

### Other information

The timestamp of the previous `MeetingDocument` must **NOT** be updated when uploading a new version of the document.

**Comment by BHenkemans** [comment]
Should it also then be that if the name of any of the versions is updated, that the name of all the versions is updated? For the sake of "Consistency"

**Comment by tomudding** [comment]
I do not think that is necessary. What I envisioned for this is the same overview we have now but with an additional 'clock'-icon if there is a history for a document. History is only about there being multiple versions of the same document.

Clicking on the 'clock'-icon opens a collapsible at the bottom of the item (in grey with some indentation) showing previous documents.

To make this flow better and also the formatting I also propose (maybe needs a separate issue) that we simplify the title of the document to only the title (incl. section number) and move the document version and date to separate fields. We then also need a 'is proceedings' checkbox for the documents that exist to enable the meeting (think eternal memorandum, etc.).
